projects/canopy/src/lib/card/card.component.spec.ts

```diff
@@ -42,50 +41,32 @@ describe('LgCardComponent', () => {
     expect(el.getAttribute('class')).toContain('lg-card');
   });
 
-  describe('when there is lg-card-header', () => {
+  describe('when there is lg-card-header, lg-card-content and lg-card-footer', () => {
```

Comment:
I think we should be testing the order of the components here as well, to make sure they appear as header first, content second and footer last.

With that in mind, is it worth having a suite of tests for when there's different combinations? E.g. a card without header and footer, a card with just a header and content? Not sure on that one 🤔

Reply:
I noticed that in the main card.component.html the template is just 
```html
<ng-content></ng-content>
```
It might be good to make it stricter...
```html
<ng-content select="lg-card-header"></ng-content>
<ng-content select="lg-card-content"></ng-content>
<ng-content select="lg-card-footer"></ng-content>
```
It's unlikely that anyone will render them in the wrong order, but feels like good practice to enforce it.

Reply:
Made the card.component template stricter as Andy suggested and also added some test for other scenarios, e.g. a card without header and footer, a card with just a header and content and a card with just the content component. Not sure here if it's not a bit too much testing, as maybe just the one test for the scenario where all of them are rendered is enough.